primitives/npos-elections/fuzzer/src/compact.rs

```diff
@@ -11,7 +11,7 @@ fn main() {
 	loop {
 		fuzz!(|fuzzer_data: &[u8]| {
 			let result_decoded: Result<InnerTestSolutionCompact, Error> =
-				<InnerTestSolutionCompact as codec::Decode>::decode(&mut &fuzzer_data[..]);
+				<InnerTestSolutionCompact as codec::Decode>::decode(&mut &*fuzzer_data);
```

Comment:
`|mut fuzzer_data: &[u8]|` and `decode(&mut fuzzer_data)` maybe?

Reply:
not this one. It's in a `fuzz!` macro that doesn't like the `mut`.